ReadMe.md

```diff
@@ -32,7 +32,8 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
 * MySQL or PostgreSQL (yet to be decided)
 
 #### Important Note : 
-    Usually MERN Stack is MongoDB , ExpressJS , ReactJS , NodeJS. But for our product, we need a hierarchical database management system not a document oriented that's why we will be using MySQL or PostgreSQL instead of MongoDB.
+
+<p> Usually MERN Stack is MongoDB , ExpressJS , ReactJS , NodeJS. But for our product, we need a hierarchical database management system not a document oriented that's why we will be using MySQL or PostgreSQL instead of MongoDB. </p>
```

Comment:
1. Remove the HTML tags please.
2. Remove the space before the comma.
3. Add the complete text that I sent you.

Reply:
1. Done
2. Not Done.
3. Not Done.